build.gradle

```diff
@@ -9,20 +9,28 @@ buildscript {
             url 'https://plugins.gradle.org/m2/'
         }
         jcenter()
+
+        maven { url 'http://developer.huawei.com/repo/' }
+
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.0.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "com.github.triplet.gradle:play-publisher:2.1.0"
         classpath "com.github.marcpoppleton:git-changelog:0.1.3"
         classpath 'com.google.android.gms:oss-licenses-plugin:0.10.2'
+        classpath 'com.huawei.agconnect:agcp:1.3.2.301'
```

Comment:
Unfortunately, I didn't find any proper solution to segregate build script class path, couldn't find any useful practise on internet. I think this should be fine, since Gradle will only download the agc plugin on a huawei build, since only huawei build flavor applies the agc plugin.